ambry-api/src/main/java/com.github.ambry/config/NettyConfig.java

```diff
@@ -110,6 +110,17 @@
   @Default("")
   public final Set<String> nettyBlacklistedQueryParams;
 
+  /**
+   * The threshold (in bytes) for POSTs via multipart/form-data.
+   * <p/>
+   * The current netty implementation cannot stream POSTs that come as multipart/form-data. It is useful to set this to
+   * reasonable number to ensure that memory usage is kept in check (i.e. protect against large blob uploads via
+   * multipart/form-data).
+   */
+  @Config("netty.multipart.post.max.size.bytes")
```

Comment:
minor. This is not really maxBlobSize. This is referring to max encoded size. Do you think we should rename it accordingly ?

Reply:
The wording or the doc doesn't mention that this is the blob size - just post size. Did you find something that indicates that this is the actual blob size?

Reply:
nope. we are good.